NAS2D/Renderer/Vector3.h

```diff
@@ -204,7 +204,7 @@ namespace NAS2D
 	template <typename BaseType>
 	bool operator>=(Vector3<BaseType> v1, Vector3<BaseType> v2)
 	{
-		return v2 <= v1;
+		return !(v1 < v2);
```

Comment:
I don't believe these two checks are equivalent. That may hold for the scalar case, but not when there are multiple component values. With multiple component values, you can get incomparable values. This is a result of the multidimensional comparison forming only a partial order.

Scalar:
`1 <= 2`, true
`!(2 < 1)`, true

Multidimensional:
`{1, 0} <= {0, 1}`, false
`!({0, 1} < {1, 0})`, true